Documentation/ceph-upgrade.md

```diff
@@ -369,15 +357,7 @@ These images are tagged in a few ways:
 
 ### Example upgrade to Ceph Octopus
 
-#### 1. Update the CRDs
```

Comment:
Updating the common.yaml and crds shouldn't actually be necessary for Ceph upgrades, I overlooked this earlier.